spectrum-vrf/src/tests.rs

```diff
@@ -0,0 +1,120 @@
+mod tests {
+    use elliptic_curve::{Group, NonZeroScalar, ProjectivePoint, Scalar};
+    use elliptic_curve::rand_core::OsRng;
+    use k256::Secp256k1;
+
+    use spectrum_crypto::digest::sha256_hash;
+    use spectrum_crypto::digest::Sha2Digest256;
+
+    use crate::vrf::{ECVRFProof, vrf_gen, vrf_prove, vrf_verify};
+
+    #[test]
+    fn normal_k256_eval() {
+        let (vrf_sk, vrf_pk) =
+            vrf_gen::<Secp256k1>().unwrap();
+
+        let m_hash: Sha2Digest256 = sha256_hash("Test_tx".as_bytes());
+
+        let proof =
+            vrf_prove::<Secp256k1>(vrf_sk, m_hash).unwrap();
+        let valid =
+            vrf_verify::<Secp256k1>(vrf_pk, m_hash.clone(), proof).unwrap();
+
+        assert_eq!(valid, true);
```

Comment:
```suggestion
        assert!(valid);
```
You can use assert! simply to test booleans